docs/data/migration/migration-data-grid-v5/migration-data-grid-v5.md

```diff
@@ -2,16 +2,20 @@
 
 <p class="description">This guide describes the changes needed to migrate the Data Grid from v5 to v6.</p>
 
-## Start using the alpha release
+## Start using the new release
 
-In `package.json`, change the version of the data grid package to `next`.
+In `package.json`, change the version of the data grid package to `latest` or `^6.0.0`.
 
 ```diff
--"@mui/x-data-grid": "latest",
-+"@mui/x-data-grid": "next",
+-"@mui/x-data-grid": "5.X.X",
++"@mui/x-data-grid": "latest",
```

Comment:
```suggestion
+"@mui/x-data-grid": "^6.0.0",
```

I think this would be preferred for most of the users to avoid accidental upgrade to v7 when it's released